R/print.data.table.R

```diff
@@ -7,6 +7,7 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
                col.names=getOption("datatable.print.colnames"),
                print.keys=getOption("datatable.print.keys"),
                trunc.cols=getOption("datatable.print.trunc.cols"),
+               trunc.char=getOption("datatable.prettyprint.char"),
```

Comment:
Hmm I didn't really mean to expose it as a new parameter. I just wanted to describe the option. If there is need to expose it I would wait for a specific FR to that end. So for now, just describing the parameter in `?print.data.table` is enough.

Reply:
well this error occurs if we don't expose it as a parameter,

```
odoc mismatches from documentation object 'print.data.table':
print.data.table
  Code: function(x, topn = getOption("datatable.print.topn"), nrows =
                 getOption("datatable.print.nrows"), class =
                 getOption("datatable.print.class"), row.names =
                 getOption("datatable.print.rownames"), col.names =
                 getOption("datatable.print.colnames"), print.keys =
                 getOption("datatable.print.keys"), trunc.cols =
                 getOption("datatable.print.trunc.cols"), quote =
                 FALSE, timezone = FALSE, ...)
  Docs: function(x, topn = getOption("datatable.print.topn"), nrows =
                 getOption("datatable.print.nrows"), class =
                 getOption("datatable.print.class"), row.names =
                 getOption("datatable.print.rownames"), col.names =
                 getOption("datatable.print.colnames"), print.keys =
                 getOption("datatable.print.keys"), trunc.cols =
                 getOption("datatable.print.trunc.cols"), trunc.char =
                 getOption("datatable.prettyprint.char"), quote =
                 FALSE, timezone = FALSE, ...)
  Argument names in docs not in code:
    trunc.char
    ```

Reply:
Right, `trunc.char` should be an argument neither in `?print.data.table` nor in the signature defined in `print.data.table.R`. Instead we should just describe how character columns are printed within the existing `?print.data.table`, and mention the option.

Reply:
Done : ) ,  added description and removed it from `alias, usage, agruments`